src/service/data_products/samples.py

```diff
@@ -159,6 +166,27 @@ class Samples(BaseModel):
     )
 
 
+@_ROUTER.get(
+    "/meta",
+    response_model=col_models.ColumnarAttributesMeta,
+    description=
+"""
+Get metadata about the samples table including column names, type,
+minimum and maximum values, etc.
+""")
+async def get_samples_meta(
+    r: Request,
+    collection_id: str = PATH_VALIDATOR_COLLECTION_ID,
+    load_ver_override: common_models.QUERY_VALIDATOR_LOAD_VERSION_OVERRIDE = None,
+    user: kb_auth.KBaseUser = Depends(_OPT_AUTH)
+) -> col_models.ColumnarAttributesMeta:
+    storage = app_state.get_app_state(r).arangostorage
+    _, load_ver = await get_load_version(storage, collection_id, ID, load_ver_override, user)
+    meta = await get_columnar_attribs_meta(storage, names.COLL_SAMPLES_META, collection_id, load_ver, load_ver_override)
```

Comment:
```suggestion
    meta = await get_columnar_attribs_meta(storage, names.COLL_SAMPLES_META, collection_id, load_ver, bool(load_ver_override))
```
Might need to split into multiple lines here